packages/jest-resolve-dependencies/src/__tests__/dependency_resolver.test.ts

```diff
@@ -7,37 +7,42 @@
  */
 'use strict';
 
-const path = require('path');
-const {normalize} = require('jest-config');
-const {buildSnapshotResolver} = require('jest-snapshot');
-const DependencyResolver = require('../index');
+import {tmpdir} from 'os';
+import path from 'path';
+import {Config} from '@jest/types';
+import {buildSnapshotResolver} from 'jest-snapshot';
+import {makeProjectConfig} from '../../../../TestUtils';
+
+import DependencyResolver from '../index';
 
 const maxWorkers = 1;
-let dependencyResolver;
+let dependencyResolver: DependencyResolver;
 let Runtime;
-let config;
-const cases = {
+let config: Config.ProjectConfig;
+const cases: {[key: string]: jest.Mock} = {
   fancyCondition: jest.fn(path => path.length > 10),
   testRegex: jest.fn(path => /.test.js$/.test(path)),
 };
-const filter = path => Object.keys(cases).every(key => cases[key](path));
+const filter = (path: Config.Path) =>
+  Object.keys(cases).every(key => cases[key](path));
 
 beforeEach(() => {
   Runtime = require('jest-runtime');
```

Comment:
should be in dev deps... sucks that we have undeclared dependencies in the tests